$OpenBSD: patch-src_ck-manager_c,v 1.6 2015/08/15 11:12:54 ajacoutot Exp $

XXX add a proper check and push upstream (adapt to OpenBSD system users naming)

--- src/ck-manager.c.orig	Sat Aug 15 11:57:59 2015
+++ src/ck-manager.c	Sat Aug 15 11:58:12 2015
@@ -1051,8 +1051,8 @@ session_is_real_user (CkSession *session,
         username = get_user_name (uid);
 
         /* filter out GDM/SDDM user */
-        if (g_strcmp0 (username, "gdm")  == 0 ||
-            g_strcmp0 (username, "sddm") == 0) {
+        if (g_strcmp0 (username, "_gdm")  == 0 ||
+            g_strcmp0 (username, "_sddm") == 0) {
                 ret = FALSE;
                 goto out;
         }
